src/lib/swankyCommand.ts

```diff
@@ -74,12 +75,12 @@ export abstract class SwankyCommand<T extends typeof Command> extends Command {
       Full command: ${JSON.stringify(process.argv)}`);
   }
 
-  protected async storeLocalConfig(projectPath: string) {
-    const localConfig : SwankyLocalConfig = {
-      node: this.swankyConfig.node,
-      contracts: this.swankyConfig.contracts
+  protected async storeConfig(projectPath: string) {
+    const configPath = process.env.SWANKY_CONFIG ?? path.resolve(projectPath, "swanky.config.json");
+    const localConfig : SwankyConfig = {
+      ...this.swankyConfig,
     }
-    await writeJSON(path.resolve(projectPath, "swanky.config.json"), localConfig, { spaces: 2 });
+    await writeJSON(configPath, localConfig, { spaces: 2 });
   }
 
   protected async storeSystemConfig() {
```

Comment:
Caution when initializing a new 'systemConfig' as outlined above. I encountered an issue where `this.swankyConfig` (the local configuration) inadvertently overwrote system accounts. This happened because of a synchronization mismatch between the local and system configurations. It's crucial to ensure that these configurations are merged correctly. Additionally, I strongly recommend conducting thorough and robust testing to verify that both configurations coexist without conflicts and that no data is lost or incorrectly overwritten during the merge process.